qemu/tests/ksm_overcommit.py

```diff
@@ -642,8 +642,8 @@ def split_parallel():
     logging.debug(utils_test.get_memory_info(lvms))
 
     # Copy ksm_overcommit_guest.py into guests
-    shared_dir = os.path.dirname(data_dir.get_data_dir())
-    vksmd_src = os.path.join(shared_dir, "scripts", "ksm_overcommit_guest.py")
+    vksmd_src = os.path.join(data_dir.get_shared_dir(),
+                             "scripts/ksm_overcommit_guest.py")
```

Comment:
Thanks for your review.  Could you please explain which way of following is better and why?

1. vksmd_src = os.path.join(data_dir.get_shared_dir(),
                                             "scripts", "ksm_overcommit_guest.py")
2. vksmd_src = os.path.join(data_dir.get_shared_dir(),
                                             "scripts/ksm_overcommit_guest.py")
3. vksmd_src = os.path.join(data_dir.get_shared_dir(),
                                              os.path.join(scripts, ksm_overcommit_guest.py))

IMO, the first one is better than the third one as join() can take more than two parameters. I don't think it is necessary to join the last two parameters first.  
@yanan-fu  could you please explain why the second one is better than the first one here as you mentioned?  Thanks!